Sources/SwiftDependencyGraphs/DependencyGraph.swift

```diff
@@ -87,11 +89,12 @@ public struct DependencyGraph<V> where V: Hashable, V: Identifiable {
     }
   }
 
-  /// A neighbour of a vertex in a graph is a vertex that is connected to it by an edge.
+  /// A neighbour of a vertex is connected to it by an edge. For `.forwards` direction, it returns
+  /// only the outgoing edges. For `.backwards` direction, it returns only the incoming edges.
   /// - Parameters:
-  ///   - vertex: The vertex
-  ///   - direction: The direction of the edge
-  /// - Returns: The neighbours of the vertex
+  ///   - vertex: The vertex for which the neighbours are computed.
+  ///   - direction: The direction of the edges that are considered.
```

Comment:
Only slight preference here, the current one is fine:

Instructive rather than descriptive:

```suggestion
  ///   - direction: The direction of edges to consider.
```

Maybe tell more: 

```suggestion
  ///   - direction: The direction of edges to consider, ignoring edges of other directions.
```